highcharts-angular/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
Why do you upgrade to angular 9 and then disable Ivy? Is this only needed for your project, or does this break with the current HighChart code?

Reply:
Hi Max! 
It's [recommended](https://angular.io/guide/creating-libraries#publishing-your-library) by Angular to turn off Ivy in production mode in order to keep the library backward compatible.

Reply:
https://angular.io/guide/ivy#ivy-and-libraries